### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

found in: github/linguist#3923

### CODE REVIEW

**Comment by stevepiercy** [comment]
This PR requires some cleanup and review before being merged. It does a good job of catching common misspellings in comments and narrative documentation, but there are some places within code where the suggested corrections are wrong.

**Comment by johansolve** [comment]
This is likely an automated commit by jsoref's spelling tool. I think we need to cherrypick it. 
But I'm a bit shocked by all my spelling errors :)

**Review by jsoref — COMMENTED** [review]
It's all manually driven. Which code things are problematic? I was somewhat concerned about queryable, but...

File: docs/source/knop_database.rst [hunk 1]
```diff
@@ -441,9 +441,9 @@ knop_database
       :param -user:
       :param string -inlinename:
 
-   .. member:: scrubKeywords(input::trait_queriable)::trait_foreach
+   .. member:: scrubKeywords(input::trait_queryable)::trait_foreach
 
-      :param trait_queriable input:
+      :param trait_queryable input:
```
Comment (stevepiercy) [inline docs/source/knop_database.rst]: These two changes should be reverted.

File: docs/source/knop_nav.rst [hunk 2]
```diff
@@ -445,9 +445,9 @@ knop_nav
    .. member:: sanitycheck()
 
 
-   .. member:: scrubKeywords(input::trait_queriable)::trait_foreach
+   .. member:: scrubKeywords(input::trait_queryable)::trait_foreach
 
-      :param trait_queriable input:
+      :param trait_queryable input:
```
Comment (stevepiercy) [inline docs/source/knop_nav.rst]: These two changes should be reverted.

File: knop8/LassoLibraries/knop.lasso [hunk 3]
```diff
@@ -8630,13 +8630,13 @@ Add -path as parameter for oncreate so the form action can be set with less conf
...
+More flexible error highlighting
 Move templates to a member tag to be make it easier to subclass (Douglas Burchard)
 Add "button". <button></button>. Subtypes are submit, reset and button. How to specify the subtype? (Douglas Burchard)
 Change ->addfield to ->insert and make ->addfield deprecated
 There is no src for input type image!
 Add ->size and ->get so the form object can be iterated
-Add -skipemtpy to to ->renderhtml
+Add -skipempty to to ->renderhtml
 Option for -> renderhtml to output without html encoding
 ->renderhtml should never html encode fields of type html
```
Comment (stevepiercy) [inline knop8/LassoLibraries/knop.lasso]: This file `knop8/LassoLibraries/knop.lasso` and all the changes in it should be reverted. It is generated automatically from the other files.

File: knop8/source/_ctype/form.inc [hunk 4]
```diff
@@ -109,9 +109,9 @@ CHANGE NOTES
 2007-01-25	JS	Added -required to ->oncreate (and a few more from ->setformat)
 2007-01-23	JS	Autogenerates id for the form itself
 2007-01-23	JS	Added ->getbutton to return the button that was clicked when submitting a form (cancel, add, save, delete)
-2007-01-23	JS	Added auto conversion of options left hand pair member to string, to make comparsions work reliably. Integer zeros don't compare nicely to strings. 
+2007-01-23	JS	Added auto conversion of options left hand pair member to string, to make comparisons work reliably. Integer zeros don't compare nicely to strings. 
 2007-01-23	JS	Added support for submit-on-enter prevention: specify -entersubmitblock at oncreate
-2007-01-19	JS	Addes renderform: -legend to be able to group form fields at render time
+2007-01-19	JS	Adds renderform: -legend to be able to group form fields at render time
```
Comment (stevepiercy) [inline knop8/source/_ctype/form.inc]: Should be "Added".

File: knop9/knoplibs/knop_database.lasso [hunk 5]
```diff
@@ -1633,7 +1633,7 @@ varname Returns the name of the variable that this type instance is stored in.
       import trait_serializable
     }
 
-	protected scrubKeywords(input::trait_queriable)::trait_forEach => {
+	protected scrubKeywords(input::trait_queryable)::trait_forEach => {
```
Comment (stevepiercy) [inline knop9/knoplibs/knop_database.lasso]: revert

File: knop9/knoplibs/knop_nav.lasso [hunk 6]
```diff
@@ -1585,7 +1585,7 @@ trace // can wait
 	scrubKeywords
 	Pinched from Kyles inline definitions. Needed to have keywords, like -path act like regular pairs
 **/
-	protected scrubKeywords(input::trait_queriable)::trait_forEach => {
+	protected scrubKeywords(input::trait_queryable)::trait_forEach => {
```
Comment (stevepiercy) [inline knop9/knoplibs/knop_nav.lasso]: revert

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please make the requested changes, and I will merge them.

**Comment by jsoref** [comment]
@stevepiercy : I believe i've addressed your concerns

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
Thank you!

**Pull request merged**